src/qgitconfig.h

```diff
@@ -62,10 +62,11 @@ namespace LibQGit2
           *
           * @param path the absolute path to the config file
           * @param priority the access priority; values with higher priority are accessed first
+          * @param repo optional repository to allow parsing of conditional includes
           *
           * @return true on success
           */
-        bool append(const QString& path, git_config_level_t level, int force);
+        bool append(const QString& path, git_config_level_t level, int force, const Repository &repo = Repository());
```

Comment:
I would put `repo` before `force` (that's the order in libgit2 as well). `force` could also have a default value, so that `repo` can still too. Also check #7 which suggests more improvements to this function's signature.

Reply:
I made `force` an optional `bool` argument and documented it. Otherwise I didn't do anything else mentioned in #7, since I don't think they are in the scope of this PR. The order of `repo` and `force` are now reversed however.